Trim the manuscript more (#449)
AlexsLemonade/OpenPBTA-manuscript: content/03.results.md

### Molecular Subtyping of OpenPBTA CNS Tumors

Since 2000, neuro-oncology experts and the WHO have collaborated to iteratively redefine central nervous system (CNS) tumor classifications [@pubmed:11895036; @doi:10.1007/s00401-007-0243-4].
In 2016 [@doi:10.1007/s00401-016-1545-1], molecular subtypes driven by genetic alterations were integrated into these classifications.
Since CBTN specimen collection began in 2011 before molecular data were classified, most tumors lacked molecular subtype information when tissue was collected.
Moreover, PBTA does not yet feature methylation arrays which are increasingly used to inform molecular subtyping and cancer diagnosis.
Therefore, we created analysis modules to systematically consider key genomic features of tumors described by the WHO in 2016 or Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011].

> Coupled with clinician and pathologist review, we generated research-grade high confidence integrated diagnoses for 60% (644/1074) of tumors  (**Table S1**) without methylation data, a major innovation of this project. <!--SAMPLECOUNT-->

Reviewer: ```suggestion
Coupled with clinician and pathologist review, we generated high confidence research-grade integrated diagnoses for 60% (644/1074) of tumors  (**Table S1**) without methylation data, a major innovation of this project. <!--SAMPLECOUNT-->
```

> We could then align OpenPBTA specimen diagnoses with WHO classifications (e.g., tumors formerly ascribed primitive neuro-ectodermal tumor [PNET] diagnoses), discover rarer tumor entities (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion), as well as identify and correct data entry errors (e.g., an embryonal tumor with multilayer rosettes (ETMR) incorrectly entered as a medulloblastoma) and histologically mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma).

Reviewer: ```suggestion
We then aligned OpenPBTA specimen diagnoses with WHO classifications (e.g., tumors formerly ascribed primitive neuro-ectodermal tumor [PNET] diagnoses), discovered rarer tumor entities (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion), as well as identified and corrected data entry errors (e.g., an embryonal tumor with multilayer rosettes (ETMR) incorrectly entered as a medulloblastoma) and histologically mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma).
```

> ](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/37ec62fdc2fd9ff157f2f2c10b69e9bb36673363/figures/pngs/figure1.png?sanitize=true){#fig:Fig1 width="7in"}

Reviewer: looks like this created a new line and needs to be updated in order to render

**Figure {@fig:Fig2}** and **Figure {@fig:S3}A** depict oncoprints recapitulating known histology-specific driver genes in primary tumors across OpenPBTA histologies, and **Table S2** summarizes all detected alterations across cancer groups.
SNV consensus calling (**Figure {@fig:S1}** and **Figure {@fig:S2}A-G**) revealed, as expected, lower tumor mutation burden (TMB) (**Figure {@fig:S2}H**) in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA) (**Figure {@fig:S2}I**), with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors [@doi:10.1016/j.cell.2017.09.048] only found within HGGs and embryonal tumors.
**Figure {@fig:Fig2}** and **Figure {@fig:S3}B** depict oncoprints recapitulating known histology-specific driver genes in primary tumors across OpenPBTA histologies, and **Table S2** summarizes all detected alterations across cancer groups.

> #### Low-grade gliomas

Author: Headers revived in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/449/commits/842cd12d22868cfacfdd839d10ae57798b8f3406.

<!--SAMPLECOUNT-->
Across HGGs, _TP53_ (57%, 36/63) and _H3F3A_ (54%, 34/63) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (29%, 18/63) which is  commonly mutated in gliomas [@doi:10.1080/14728222.2018.1487953].
We observed recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors leverage multiple oncogenic mechanisms to activate tyrosine kinases, as previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w].
GSVA showed upregulation (ANOVA Bonferroni-corrected p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**).
The two ultra-hypermutated tumors (> 100 Mutations/Mb) were from patients with mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].

<!--SAMPLECOUNT-->

> Considering embryonal tumors, 25% (15/60) of ependymomas were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive [@doi:10.1038/nature13109], and 68% (21/31) of craniopharyngiomas contained _CTNNB1_ mutations (**Figure {@fig:Fig2}D**).

Reviewer: These are the "other" not embryonals, so this should be changed.

> Considering embryonal tumors, 25% (15/60) of ependymomas were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive [@doi:10.1038/nature13109], and 68% (21/31) of craniopharyngiomas contained _CTNNB1_ mutations (**Figure {@fig:Fig2}D**).

Reviewer: ```suggestion
Considering other tumors, 25% (15/60) of ependymomas were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive [@doi:10.1038/nature13109], and 68% (21/31) of craniopharyngiomas contained _CTNNB1_ mutations (**Figure {@fig:Fig2}D**).
```

_FGFR1_ and _PIK3CA_ mutations significantly co-occurred in LGGs (OR = 77.25, 95% CI: 10.0 - 596.8, q = 3.12e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; @doi:10.1186/s40478-020-01027-z].
Of HGG tumors with _TP53_ or _PPM1D_ mutations, 53/55 (96.3%) had mutations in only one of these genes (OR = 0.17, 95% CI: 0.04 - 0.89, q = 0.056), recapitulating previous observations that these mutations are usually mutually exclusive in HGGs [@https://doi.org/10.1038/ng.2938].

CNV and SV analyses revealed that HGG, DMG, and medulloblastoma tumors has the most unstable genomes, while craniopharyngiomas and schwannomas generally lacked somatic CNV (**Figure {@fig:S3}C**).
These CNV patterns largely aligned with our TMB estimates (**Figure {@fig:S2}H**).
SV and CNV breakpoint densities were significantly correlated (linear regression p = 1.05e-38; **Figure {@fig:Fig3}C**), and as expected, the number of chromothripsis regions called increased with breakpoint density (**Figure {@fig:S3}D-E**).

> We identified chromothripsis events in 31% (N = 12/39) of DMGs and in 44% (N = 21/48) of other HGGs (**Figure {@fig:Fig3}D**), and we found evidence of chromothripsis in over 15% of sarcomas, PXAs, metastatic secondary tumors, chordomas, glial-neuronal tumors, germinomas, meningiomas, ependymomas, medulloblastomas, ATRTs, and other embryonal tumors.

Reviewer: ```suggestion
We identified chromothripsis events in 31% (N = 12/39) of DMGs and in 44% (N = 21/48) of other HGGs (**Figure {@fig:Fig3}D**), and found evidence of chromothripsis in over 15% of sarcomas, PXAs, metastatic secondary tumors, chordomas, glial-neuronal tumors, germinomas, meningiomas, ependymomas, medulloblastomas, ATRTs, and other embryonal tumors.
```

### Transcriptomic Landscape of Pediatric Brain Tumors

Most RNA-Seq samples in the PBTA were prepared with ribosomal RNA depletion followed by stranded sequencing (N = 977), while remaining samples were prepared with poly-A selection (N = 58).
Since batch correction was not feasible (see **Limitations of the Study** and **Figure {@fig:S7}A**), the following transcriptomic analyses considered only stranded samples.

#### Prediction of _TP53_ oncogenicity and telomerase activity

> We applied TCGA-trained classifier [@doi:10.1016/j.celrep.2018.03.076] to calculate a _TP53_ score, a proxy for _TP53_ gene or pathway dysregulation, and subsequently infer tumor _TP53_ inactivation status.

Reviewer: ```suggestion
We applied a TCGA-trained classifier [@doi:10.1016/j.celrep.2018.03.076] to calculate a _TP53_ score, a proxy for _TP53_ gene or pathway dysregulation, and subsequently infer tumor _TP53_ inactivation status.
```

> We identified "true positive" _TP53_ alterations from high-confidence SNVs, CNVs, SVs, and fusions in _TP53_, annotating tumors as "activated" if they harbored one of p.R273C or p.R248W gain-of-function mutations [@doi:10.1038/ng0593-42], or "lost" if 1) the given patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, 2) the tumor harbored a known hotspot mutation, or 3) the tumor contained two hits (e.g. both SNV and CNV), suggesting both alleles were affected.

Reviewer: ```suggestion
We identified "true positive" _TP53_ alterations from high-confidence SNVs, CNVs, SVs, and fusions in _TP53_, annotating tumors as "activated" if they harbored one of p.R273C or p.R248W gain-of-function mutations [@doi:10.1038/ng0593-42], or "lost" if 1) the patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, 2) the tumor harbored a known hotspot mutation, or 3) the tumor contained two hits (e.g. both SNV and CNV), suggesting both alleles were affected.
```

If the _TP53_ mutation did not reside within the DNA-binding domain or no alterations in _TP53_ were detected, we annotated the tumor as "other," indicating an unknown _TP53_ alteration status.
The classifier achieved a high accuracy (AUROC = 0.86) for rRNA-depleted, stranded tumors, but it did not perform as well on the poly-A tumors in this cohort (AUROC = 0.62; **Figure {@fig:S5}A**).

> We observed that "activated" and "lost" tumors had similar _TP53_ scores (**Figure {@fig:Fig4}B**, Wilcoxon p = 0.92), contrasting our expectation that "activated" tumors would have higher _TP53_ scores.

Reviewer: ```suggestion
We observed that "activated" and "lost" tumors had similar _TP53_ scores (**Figure {@fig:Fig4}B**, Wilcoxon p = 0.92), contrasting our expectation that "lost" tumors would have higher _TP53_ scores.
```

Reviewer: We thought this score reflected inactivation of TP53

This difference suggests that classifier scores > 0.5 may actually represent an oncogenic, or altered, _TP53_ phenotype rather than solely _TP53_ inactivation, as interpreted previously [@doi:10.1016/j.celrep.2018.03.076].
However, "activated" tumors showed higher _TP53_ expression compared to those with _TP53_ "loss" mutations (Wilcoxon p = 0.006, **Figure {@fig:Fig4}C**).
DMGs, medulloblastomas, HGGs, DNETs, ependymomas, and craniopharyngiomas, all known to harbor _TP53_ mutations, had the highest median _TP53_ scores (**Figure {@fig:Fig4}D**).
By contrast, gangliogliomas, LGGs, meningiomas, and schwannomas had the lowest median scores.

> We hypothesized that tumors from patients with LFS (N = 8) would have higher _TP53_ scores, which we indeed observed for 8/10 tumors from LFS patients (**Table S3**).

Reviewer: ```suggestion
We hypothesized that tumors (N = 10) from patients with LFS (N = 8) would have higher _TP53_ scores, which we indeed observed for 8/10 tumors (**Table S3**).
```

> Although two tumors from LFS patients had low _TP53_ scores (`BS_DEHJF4C7` at 0.09 and `BS_ZD5HN296` at 0.28), pathology reports confirmed that both patients were diagnosed with LFS with a _TP53_ pathogenic germline variant.

Reviewer: ```suggestion
Although two tumors had low _TP53_ scores (`BS_DEHJF4C7` at 0.09 and `BS_ZD5HN296` at 0.28), pathology reports confirmed that both patients were diagnosed with LFS and harbored a _TP53_ pathogenic germline variant.
```

These two LFS tumors also had low tumor purity (16% and 37%, respectively), suggesting that accurate classification may require a certain level of tumor content.

> We suggest that this classifier could be generally applied to infer _TP53_ function in the absence of a predicted oncogenic _TP53_ alteration or DNA sequencing.

Reviewer: ```suggestion
We suggest this classifier could be generally applied to infer _TP53_ function in the absence of a predicted oncogenic _TP53_ alteration or DNA sequencing.
```

> We suggest that this classifier could be generally applied to infer _TP53_ function in the absence of a predicted oncogenic _TP53_ alteration or DNA sequencing.

Reviewer: This sentence is something new we found with this paper, so I would suggest leaving it.

Three of these patients harbored pathogenic germline variants in one of the aforementioned genes.
While we did not detect a _known_ pathogenic variant in the germline of PT_VTM2STE3, this patient's pathology report contained a self-reported _PMS2_ variant, and we indeed found 19 intronic variants of unknown significance (VUS) in their _PMS2_.
This is not surprising since an estimated 49% of germline _PMS2_ variants in patients with CMMRD and/or Lynch syndrome are VUS [@doi:10.1136/jmedgenet-2020-107627].
Interestingly, while the cell line derived from patient PT_VTM2STE3's tumor at progression was not hypermutated (TMB = 5.7 Mut/Mb), it only contained the MMR2 signature, suggesting selective pressure to maintain a mismatch repair (MMR) phenotype _in vitro_.
Only one of the two cell lines derived from patient PT_JNEV57VK's progressive tumor was hypermutated (TMB = 35.9 Mut/Mb).

> Their hypermutated cell line was strongly weighted towards signature 11, while their non-hypermutated cell line showed several lesser signature weights (1, 11, 18, 19, MMR2; **Table S2**).

Reviewer: ```suggestion
The hypermutated cell line was strongly weighted towards signature 11, while the non-hypermutated cell line showed several lesser signature weights (1, 11, 18, 19, MMR2; **Table S2**).
```

I would take out "their" as we prefer to separate the patient from the tumor or cell line in this instance. It is the cancer's genetics here reflected in the cell line, not the patient's.

Reviewer 2: That answers my question in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/449#discussion_r1157826080.

Author: This change was implemented via a different suggestion - https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/449/commits/9a309829020d3362dabd60902e141b7ec314b5a1

This mutational process plasticity highlights the importance of careful genomic characterization and model selection for preclinical studies.

Signature 18, which has been associated with high genomic instability and can induce a hypermutator phenotype [@doi:10.1038/s43018-020-0027-5], was uniformly represented among hypermutant solid tumors.

> Additionally, all hypermutant HGG tumors or cell lines had dysfunctional _TP53_ (**Table {@tbl:Table2}**), consistent with previous findings that tumors with high genomic instability depend on _TP53_ dysregulation [@doi:10.1038/s43018-020-0027-5].

Reviewer: ```suggestion
Additionally, all hypermutant HGG tumors or cell lines had dysfunctional _TP53_ (**Table {@tbl:Table2}**), consistent with previous findings that tumors with high genomic instability signatures require _TP53_ dysregulation [@doi:10.1038/s43018-020-0027-5].
```
clarifying this

Table: **Patients with hypermutant tumors.** Listed are patients with at least one hypermutant or ultra-hypermutant tumor or cell line. Pathogenic (P) or likely pathogenic (LP) germline variants, coding region TMB, phase of therapy, therapeutic interventions, cancer predisposition (CMMRD = Constitutional mismatch repair deficiency), and molecular subtypes are included. {#tbl:Table2}

Next, we asked whether transcriptomic classification of _TP53_ dysregulation and/or telomerase activity recapitulate these oncogenic biomarkers' known prognostic influence.

> We identified several expected trends, including a significant overall survival benefit following full tumor resection (HR = 0.35, 95% CI = 0.2 - 0.62, p < 0.001) or if the tumor was an LGG (HR = 0.046, 95% CI = 0.0062 - 0.34, p = 0.003), and a significant risk if the was an HGG (HR = 6.2, 95% CI = 4.0 - 9.5, p < 0.001) (**Figure {@fig:Fig4}F**; **STAR Methods**).

Reviewer: ```suggestion
We identified several expected trends, including a significant overall survival benefit following full tumor resection (HR = 0.35, 95% CI = 0.2 - 0.62, p < 0.001) or if the tumor was an LGG (HR = 0.046, 95% CI = 0.0062 - 0.34, p = 0.003), and a significant risk if the was a HGG (HR = 6.2, 95% CI = 4.0 - 9.5, p < 0.001) (**Figure {@fig:Fig4}F**; **STAR Methods**).
```

I always get confused about article usage here - I read this as "was a high-grade glioma" myself, but I get if someone reads as "was an HGG:

Reviewer 2: "An" is correct because HGG, when pronounced H-G-G, starts with a vowel sound. (My argument hinges on my assumptions about the pronunciation.)

Reviewer 2: Missing a word:
```suggestion
We identified several expected trends, including a significant overall survival benefit following full tumor resection (HR = 0.35, 95% CI = 0.2 - 0.62, p < 0.001) or if the tumor was an LGG (HR = 0.046, 95% CI = 0.0062 - 0.34, p = 0.003), and a significant risk if the tumor was an HGG (HR = 6.2, 95% CI = 4.0 - 9.5, p < 0.001) (**Figure {@fig:Fig4}F**; **STAR Methods**).
```

High telomerase scores were associated with poor prognosis across brain tumor histologies (HR = 20, 95% CI = 6.4 - 62, p < 0.001), demonstrating that `EXTEND` scores calculated from RNA-Seq are an effective rapid surrogate measure for telomerase activity.
Higher _TP53_ scores were associated with significant survival risks (**Table S4**) within DMGs (HR = 6436, 95% CI = 2.67 - 1.55e7, p = 0.03) and ependymomas (HR = 2003, 95% CI = 9.9 - 4.05e5, p = 0.005).

> Given this result, we next assessed whether different HGG molecular subtypes carry different survival risks.

Reviewer: ```suggestion
Given this result, we next assessed whether different HGG molecular subtypes carry different survival risks if stratified by _TP53_ status.
```

> We found that DMG H3 K28 tumors with _TP53_ loss had significantly worse prognosis (HR = 2.8, CI = 1.4-5.6, p = 0.003) than did DMG H3 K28 tumors with wildtype _TP53_ (**Figure {@fig:Fig4}G** and **Figure {@fig:Fig4}H**), reflecting results from two recent restrospective analyses of DIPG tumors [@doi:10.1158/1078-0432.CCR-22-0803; @doi:10.1007/s11060-021-03890-9].

Reviewer: ```suggestion
We found that DMG H3 K28 tumors with _TP53_ loss had significantly worse prognosis (HR = 2.8, CI = 1.4-5.6, p = 0.003) than those with wildtype _TP53_ (**Figure {@fig:Fig4}G** and **Figure {@fig:Fig4}H**), recapitulating results from two recent restrospective analyses of DIPG tumors [@doi:10.1158/1078-0432.CCR-22-0803; @doi:10.1007/s11060-021-03890-9].
```

Although CD8+ T-cell infiltration across all cancer groups was minimal (**Figure {@fig:Fig5}C**), we observed signal in specific cancer molecular subtypes (Groups 3 and 4 medulloblastoma) as well as outlier tumors (BRAF-driven LGG, BRAF-driven and wildtype ganglioglioma, and CNS embryonal NOS; **Figure {@fig:S6}E**)
Surprisingly, the classically immunologically-cold HGGs and DMGs [@doi:10.1186/s40478-018-0553-x; @doi:10.1093/brain/awab155] contained higher overall fractions of immune cells, primarily monocytes, dendritic cells, and NK cells (**Figure {@fig:Fig5}C**).
Thus, quanTIseq might actually have captured microglia within these immune cell fractions.

While we did not detect notable prognostic effects of immune cell infiltration on overall survival in HGGs or DMGs, we found that high levels of macrophage M1 and monocytes were associated with poorer overall survival (monocyte HR = 2.1e18, 95% CI = 3.80e5 - 1.2e31, p = 0.005, multivariate Cox) in medulloblastomas (**Figure {@fig:Fig5}D**).
We further reproduced previous findings (**Figure {@fig:Fig5}E**) that medulloblastomas typically have low expression of _CD274_ (PD-L1) [@doi:10.18632/oncotarget.24951].

> We also found that higher expression of _CD274_ was significantly associated with improved overall prognosis for medulloblastoma tumors, although only marginally (HR = 0.0012, 95% CI = 7.5e−06 - 0.18, p = 0.008, multivariate Cox) (**Figure {@fig:Fig5}D**).

Reviewer: ```suggestion
We also found that higher expression of _CD274_ was significantly associated with improved overall prognosis for medulloblastoma tumors, although marginal (HR = 0.0012, 95% CI = 7.5e−06 - 0.18, p = 0.008, multivariate Cox) (**Figure {@fig:Fig5}D**).
```
?

This result may be explained by the higher expression of _CD274_ observed in WNT subtype tumors by us and others [@doi:10.1080/2162402X.2018.1462430], as this diagnosis carries the best prognosis of all medulloblastoma subgroups (**Figure {@fig:Fig5}E**).

We additionally explored the ratio of CD8+ to CD4+ T cells across tumor subtypes.
This ratio has been associated with better immunotherapy response and prognosis following PD-L1 inhibition in non-small cell lung cancer or adoptive T cell therapy in multiple stage III or IV cancers [@doi:10.1136/jitc-2021-004012; @doi:10.4236/jct.2013.48164].
While adamantinomatous craniopharyngiomas and Group 3 and Group 4 medulloblastomas had the highest ratios (**Figure {@fig:S6}F**), very few tumors had ratios greater than 1, highlighting an urgent need to identify novel therapeutics for pediatric brain tumors with poor prognosis.

> Finally, we explored the potential influence of tumor purity by repeating selected transcriptomic analyses restricted to samples with high tumor purities within their cancer group(see **STAR Methods**).

Reviewer: How about

>Finally, we explored the potential influence of tumor purity by repeating selected transcriptomic analyses restricted to samples with high tumor purity (see **STAR Methods**).
